### ISSUE TITLE
[Bug] liquid-c not found

### ISSUE THREAD

**Issue by MJochim** [daviddarnes/alembic#118]
**Please read the [contributing guidelines](https://github.com/daviddarnes/alembic/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/daviddarnes/alembic/blob/master/CODE_OF_CONDUCT.md) before creating an issue.**

*As an aside: The links above are broken. Those that appear to the right while writing this issue (“it looks like this is your first time opening an issue in this project …” work fine.*

After the upgrade to 3.0.7, I cannot build my Alembic-themed jekyll sites anymore. I tried on an Arch Linux machine (ruby 2.5.3p105) and within an Alpine-based docker container (don't know the Ruby version there). I always get this error:

```
  Dependency Error: Yikes! It looks like you don't have liquid-c or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- liquid-c' If you run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
```

I can reproduce this using the following minimal example with only two files:

**Gemfile**:

```
source "https://rubygems.org"
gem "jekyll"
gem "alembic-jekyll-theme"
```

**_config.yml**

```
theme: alembic-jekyll-theme
```

`bundle install && bundle exec jekyll serve` results in the above error. Adding `gem "liquid-c"` to Gemfile does not help. I am not a Ruby guy, I may have missed something I need to do. But to me, this looks like a bug in the theme. It would be great if you could take a look!

**Comment by daviddarnes** [comment]
Hey @MJochim, sorry you're having trouble with the new version of Alembic. Seems odd that it's not installing liquid-c, the reference for the gem is right here https://github.com/daviddarnes/alembic/blob/master/alembic-jekyll-theme.gemspec#L29. I'll have a go installing a fresh to see if I get the error. Have you tried installing outside the docker container as a plain Jekyll project? Also, what version of Jekyll are you using?

**Comment by MJochim** [comment]
Yes, the Arch Linux machine is non-docker. I'm using Jekyll 3.8.5:

```
$ bundle exec jekyll --version
jekyll 3.8.5
$ jekyll --version
jekyll 3.8.5
$ ruby --version
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
```

Just now I deleted liquid-c manually (`$GEM_HOME/gems/liquid-c-4.0.0` and `$GEM_HOME/specifications/liquid-c-4.0.0.gemspec`) and then re-executed `bundle install && bundle exec jekyll serve`. The result is:

```
[…]
Using jekyll-sitemap 0.13.0
Using gemoji 3.0.0
Using jemoji 0.10.2
Fetching liquid-c 4.0.0
Installing liquid-c 4.0.0 with native extensions
Using alembic-jekyll-theme 3.0.7
Using bundler 1.17.1
Bundle complete! 2 Gemfile dependencies, 48 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
```

So it does re-install liquid-c (and I can confirm that by looking at `$GEM_HOME/gems/liquid-c`).

But the error message from `bundle exec jekyll serve` remains.

**Comment by osheroff** [comment]
having the same problem.  downgrading to 3.0.6 fixes.

**Comment by daviddarnes** [comment]
Well it's nice to know people are keeping up to date and using the theme, but sad I've broken your stuff :(. I've released 3.0.8 which removes liquid-c entirely, could you report back if it's working? Thanks all!

**Comment by MJochim** [comment]
Well, thank you for your exceptionally fast response - and for the theme in the first place :-).

Upgrading to 3.0.8 works perfectly.
